### PR TITLE
[SV DateTimeV2] Swedish datetime support - code skeleton

### PR DESCRIPTION
**Pull request by pete1854**
Adding support for datetime in swedish

### CODE REVIEW

**Comment by tellarin** [comment]
Hi @pete1854 I suggest making a PR just with the code skeleton for Swedish (just not registering the models) and then a PR with the translated specs. Then we go little by little by subtype refinining things.
Can you convert this PR in one for the skeleton (basically the structure copied from English with renames and some configs for Swedish)? This way we can discuss issues quickly and merge as frequently as possible.

**Review by tellarin — CHANGES_REQUESTED** [review]
There are only some minor issue before we can merge this skeleton.
Please change the PR from draft to regular once the changes are in. Thanks.

File: .NET/Microsoft.Recognizers.Text.DataDrivenTests/DateTime/TestDateTime_Swedish.cs [hunk 1]
```diff
@@ -1,263 +0,0 @@
...
-        {
-            Extractors = new Dictionary<string, IDateTimeExtractor>();
-            Parsers = new Dictionary<string, IDateTimeParser>();
-        }
-
-        [NetCoreTestDataSource]
-        [TestMethod]
-        public void DateExtractor(TestModel testSpec)
-        {
-            ExtractorInitialize(Extractors);
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.DataDrivenTests/DateTime/TestDateTime_Swedish.cs]: I'd add this file back with these tests also commented out.

File: .NET/Microsoft.Recognizers.Text.sln [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31402.337
+# Visual Studio Version 17
```
[the remark this answers is not in the document]
Comment (pete1854) [reply .NET/Microsoft.Recognizers.Text.sln:3]: Sorry, my bad. I forgot to pay attention to this. Will correct.

**Comment by pete1854** [comment]
Apart from the two indentation issues which I haven't been able to correct (since they do not exist in my code) this shouild be OK now.

**Review by tellarin — APPROVED** [review]
I've fixed the remaining identiation issues (tab vs whitespaces).

**Pull request merged**